src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/UsePropertiesWhereAppropriateTests.cs

```diff
@@ -331,7 +332,7 @@ public int GetSomethingWithUnboundInvocation()
 ");
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/7222")]
+        [Fact]
```

Comment:
Shall I keep the `Skip`?

Reply:
@mavasani ?

Reply:
Ping @mavasani

Reply:
@mavasani Could you please let me know if that's ok to remove the `skip`?

Reply:
If the test is passing it is fine to remove the skip attribute - thanks!